data/wanderer/wanderers start.txt

```diff
@@ -2721,19 +2721,47 @@ mission "Wanderers Ap'arak 2"
 
 
 
+# TODO: When "complete anywhere" functionality is available, 
+# condense the following three missions into a single mission
 mission "Wanderers Ap'arak 3: Pug"
 	landing
 	invisible
 	source "Varu Tev'kei"
 	to offer
 		has "Wanderers Ap'arak 2: done"
-	npc kill
+	npc save
 		personality staying unconstrained uninterested
 		government "Pug (Wanderer)"
 		ship "Pug Arfecta" "Hork Ekekek Ptui"
+	to complete
+		never
+
+mission "Wanderers Ap'arak 3: Pug 2"
+	landing
+	invisible
+	source "Varu Tev'kei"
+	to offer
+		has "Wanderers Ap'arak 2: done"
+		not "Wanderers Ap'arak 3: done"
```

Comment:
I think this is incorrect for existing pilots, and will spawn an additional Arfectas.

It should be an "offered" check instead, and the first mission should be both renamed and also given this check. To maintain compatibility, the penalty mission needs to set this particular "done" condition.

Reply:
It seems like it would only spawn an additional Arfecta if someone had a save between Ap'arak 2 and 3, and I tested on my 0.9.13 pilot with no additional Arfecta spawns. I'll tighten up the requirements as much as possible, but it seems unlikely players will have a specific save there unless they're actively following this PR.

Reply:
Honestly, I had completely missed that this was for a different mission than the one it was replacing, which was designed to generally not be "done."

The change to use "offered" was still necessary, but indeed much less impactful than my original perception.